src/components/ActiveTeamsComponent.jsx

```diff
@@ -11,84 +9,69 @@ const ActiveTeamsComponent = ({
   handleDragStart,
   handleDragOver,
   handleDragDrop,
-}) => {
-  const myListRef = useRef(null);
-  const [isEmpty, setIsEmpty] = useState(true);
-
-  useEffect(() => {
-    if (myListRef.current === null) {
-      setIsEmpty(true);
-    } else if (myListRef.current.getElementsByTagName('li').length < 1) {
-      setIsEmpty(true);
-    } else {
-      setIsEmpty(false);
-    }
-  }, [teams]);
+}) => (
+  <div style={{ padding: '10px', border: '1px solid black' }}>
+    <h2>Active teams</h2>
+    <div
+      onDragOver={handleDragOver}
+      onDrop={handleDragDrop}
+      id="active"
+    >
+      {teams.length < 1 && (
+        <div
+          style={{ border: '2px solid green', width: '30vw' }}
+          id="active-1"
+        >
+          Drop here
+        </div>
+      )}
 
-  return (
-    <div style={{ padding: '10px', border: '1px solid black' }}>
-      <h2>Active teams</h2>
-      <div
-        onDragOver={handleDragOver}
-        onDrop={(e) => handleDragDrop(e)}
-        id='active'
-      >
-        {isEmpty && (
-          <div
-            style={{ border: '2px solid green', width: '30vw' }}
-            id='active-1'
-          >
-            Drop here
-          </div>
-        )}
-        {teams.length > 0 && (
-          <ul ref={myListRef}>
-            {teams.map(
-              (team, index) =>
-                team.active && (
-                  <li key={index} draggable>
-                    <button
-                      type='button'
-                      onDragStart={handleDragStart(index)}
-                      onDragOver={handleDragOver}
-                    >
-                      Drag
-                    </button>
-                    <input
-                      type='text'
-                      value={team.name}
-                      onChange={(e) => handleNameInput(e, index)}
-                    />
-                    <button
-                      type='button'
-                      onClick={() => handleScoreDecrease(index)}
-                    >
-                      -
-                    </button>
-                    <span>{teams[index].score}</span>
-                    <button
-                      type='button'
-                      onClick={() => handleScoreIncrease(index)}
-                    >
-                      +
-                    </button>
-                    <button type='button' onClick={() => handleTransfer(index)}>
-                      Transfer
-                    </button>
-                    <button type='button' onClick={() => handleDelete(index)}>
-                      Delete
-                    </button>
-                  </li>
-                ),
-            )}
-          </ul>
-        )}
-      </div>
-      <button type='button' onClick={() => handleAddTeam('active')}>
-        Add team
-      </button>
+      {teams.length > 0 && (
+        <ul>
+          {teams.map(
```

Comment:
This arrow function will not be saved by React and would be re-created on every render. Usually it's more efficient to store it in a separate render function. Something like:
`{teams.map(renderTeam)}`
This makes the code more readable as you're describing the code by the function name. Another elegant approach would be to create a separat component out of it. This would even help you to reduce the amount of duplicated code.

Reply:
True, because I'm using the same in the EliminatedTeam component too. Possibly the latter suggestion will be better here. Thanks @iteratus